src/Pages/Employer/BrowseFreelancersPage.jsx

```diff
@@ -79,8 +79,8 @@ const BrowseFreelancersPage = () => {
       const params = {
         page: page,
         per_page: 8,
-        category:categoryFilter,
-        location:locationFilter,
+        category:categoryFilter.join(",").replace(/\s+/g, "_").toUpperCase(),
+        location:locationFilter.join(",").replace(/\s+/g, "_").toUpperCase(),
```

Comment:
This might be incorrect. You should explicitly create this mapping between value obtained from checkbox and the values accepted by backed. 

REF: https://api.dev.grull.tech/docs